Force MonoDevelop to use XULRunner

This changes configure to set MOZILLA_HOME to XULRunner - it is propogated to
/usr/bin/monodevelop. This allows MonoDevelop to be run without Firefox.
--- /home/paul/conary/builds/monodevelop/monodevelop-0.14/configure	2007-06-12 17:02:24.000000000 +0100
+++ configure	2007-06-14 08:36:27.000000000 +0100
@@ -5690,36 +5690,18 @@
 
 #We need to know where Mozilla is for launch scripts and possible for AspNetEdit
 #Check pkg-config first
-MOZILLA_HOME="`$PKG_CONFIG --variable=libdir mozilla-gtkmozembed`"
-if !(test -n "$MOZILLA_HOME"); then
-	MOZILLA_HOME="`$PKG_CONFIG --variable=libdir firefox-gtkmozembed`"
-fi
-if !(test -n "$MOZILLA_HOME"); then
-	MOZILLA_HOME="`$PKG_CONFIG --variable=libdir xulrunner-gtkmozembed`"
-fi
+MOZILLA_HOME="`$PKG_CONFIG --variable=libdir xulrunner-gtkmozembed`"
 
 #if MOZILLA_HOME is still empty, try to detect without pkg_config
 #(NOTE: any changes to this block should be kept in sync with the one in monodevelop.in)
 if !(test -n "$MOZILLA_HOME"); then
-	if test -n "$MOZILLA_FIVE_HOME"; then
-		MOZILLA_HOME=$MOZILLA_FIVE_HOME
-	elif grep GRE_PATH /etc/gre.d/*.conf > /dev/null ; then
-		MOZILLA_HOME=$(grep -h GRE_PATH= /etc/gre.d/*.conf | cut -d '"' -f 2 -d = | head -n 1)
-	elif  $(which mozilla 2>/dev/null)  && grep MOZILLA_FIVE_HOME= "$(which mozilla)" > /dev/null ; then
-		MOZILLA_HOME=$(grep MOZILLA_FIVE_HOME= $(which mozilla) | cut -d '"' -f 2 | cut -d '=' -f 2 )
-	elif  $(which firefox 2>/dev/null)  && grep MOZILLA_FIVE_HOME= "$(which firefox)" > /dev/null ; then
-		MOZILLA_HOME=$(grep MOZILLA_FIVE_HOME= $(which firefox) | cut -d '"' -f 2 | cut -d '=' -f 2 )
-	elif  $(which firefox 2>/dev/null)  && grep MOZILLA_LIBDIR= "$(which firefox)" > /dev/null ; then
-		MOZILLA_HOME=$(grep MOZILLA_LIBDIR= $(which firefox) | cut -d '"' -f 2 | cut -d '=' -f 2)
-	else
-		if test "x$enable_aspnetedit" = "xyes"; then
-			{ { echo "$as_me:$LINENO: error: Cannot detect Mozilla library directory. AspNetEdit addin cannot be built." >&5
+	if test "x$enable_aspnetedit" = "xyes"; then
+		{ { echo "$as_me:$LINENO: error: Cannot detect Mozilla library directory. AspNetEdit addin cannot be built." >&5
 echo "$as_me: error: Cannot detect Mozilla library directory. AspNetEdit addin cannot be built." >&2;}
-   { (exit 1); exit 1; }; }
-		else
-			{ echo "$as_me:$LINENO: WARNING: Cannot detect Mozilla library directory. WelcomePage addin may not be able to function." >&5
+{ (exit 1); exit 1; }; }
+	else
+		{ echo "$as_me:$LINENO: WARNING: Cannot detect Mozilla library directory. WelcomePage addin may not be able to function." >&5
 echo "$as_me: WARNING: Cannot detect Mozilla library directory. WelcomePage addin may not be able to function." >&2;}
-		fi
 	fi
 fi
 
